### ISSUE TITLE
Show Committed quantity on Supplier item

### ISSUE THREAD

**Issue by ananya-rajput** [dailykit/dailyos#8]
As a restaurant owner, the user should always be able to see the committed quantity from the on hand quantity.

What is committed quantity?
The committed quantity is the quantity committed or assigned to a work order.

How will the committed quantity calculated:
On a work order, the user has added the quantity for a work.
This committed quantity is calculated as how much quantity had to be taken out from an item to create the work order quantity.
Now why should these two quantities differ?
Now, we have two type of work orders:
**1. work order for derivative processing**: This work order is for to create a derivative processing from the shipped processing.
Creating a new processing from an existing one has yield factor. For. eg: creating mashed potato from raw potato results in removing the skin and hence the quantity of the final product differs from the initial product.

How to calculate the committed quantity from the quantity added as an output for a work order?

Let's say Alex needs 200g of Mashed potato from the supplier item - Raw potato. And the yield percentage of mashed potato from raw potato is 85%. 
Now what Alex needs to know is how much have has been committed for mashed potato work order from my raw potato on hand quantity:

**So the output quantity of the work order, x** = 200gm
**The yield percentage of the mashed potato for raw potato, y**  = 85
Hence, **the committed quantity (let's say:a )** of the raw potato, **a = x*100/y** = 200*100/85 = 235gm.

**2. work order for planned sachet**: This work order is for to create planned lot sachets from a bulk item.
Now, creating sachets of an item, only needs portioning which requires no yield. Hence, in this case the output quantity of a work order equals to the quantity needs to be committed.
i.e. **a=x**
Note: The output quantity, x = quantity of one sachet * number of sachets

So in summary:
Calculating committed quantity:
**1. from work order of derivative processing** : a = x*100/y
**2. from work order of planned lot sachets** : a = x
